exporter_test.go

```diff
@@ -639,6 +639,72 @@ version = "4.5.6"
 				})
 			})
 
+			when("checking the image manifest", func() {
```

Comment:
I wonder if we could pull this test out of the `previous image exists` contexts and put it at the top level, as the functionality being tested doesn't depend on the previous image existing or not.

Reply:
Please see https://github.com/buildpacks/lifecycle/pull/498/commits/e32d468109f145d40f80ec070f7300d3f807ed80